src/core/primitives/RovingFocusGroup/stories/RovingFocusGroup.stories.tsx

```
@@ -0,0 +1,29 @@
+import React from 'react';
+import SandboxEditor from '~/components/tools/SandboxEditor/SandboxEditor';
+import RovingFocusGroup from '../RovingFocusGroup';
+
+// More on how to set up stories at: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+
+const items = [
+    {title: "1", content: "RovingFocusGroup"},
```

ESLint

[error] A space is required after '{'.

(object-curly-spacing)